manifests/params.pp

```diff
@@ -86,6 +87,8 @@
   $env_config_path            = '/etc/rabbitmq/rabbitmq-env.conf'
   $erlang_cookie              = undef
   $interface                  = 'UNSET'
+  $mqtt_ensure                = false
```

Comment:
Should this be an actual `ensure` value? (Eg `present` or `absent`, or otherwise `undef`)